src/types.ts

```diff
@@ -73,6 +73,16 @@ export interface IdentityOptions {
 	auth: string | ((client: Client) => string) | (() => Promise<string>);
 }
 
+/**
+ *
+ */
+export interface ClientSettings {
+	/**
+	 * Disabled by default. If set to true, chat will be logged to console using console.log
+	 */
+	logToConsole?: boolean;
```

Comment:
A more conventional logger with multiple levels would be preferred.

Reply:
Agreed. Pull request should not be accepted until this has been properly implemented.